jobs/README.md

```diff
@@ -1,22 +1,28 @@
 # DevStaff Community Job Openings
 
-In this section you can find some job openings listed by our community members. If you want to add a new job opening do it at the top of the list below, git commit, git push, and create a pull request on GitHub.
+In this section you can find some job openings listed by our community members.
+If you want to add a new job opening do it at the top of the list below, git
+commit, git push, and create a pull request on GitHub.
 
-__Be advised that Job Openings older than 3 months will be removed for the sake of keeping current. Feel free to repost.__
+__Be advised that Job Openings older than 3 months will be removed for the sake
+of keeping current. Feel free to repost.__
```

Comment:
maybe change this to: "of keeping the board up-to-date"

Reply:
Done. Thanks!